cmake/OpenCVFindLibsGrfmt.cmake

```diff
@@ -30,6 +31,7 @@ if(WITH_JPEG)
   if(BUILD_JPEG)
     ocv_clear_vars(JPEG_FOUND)
   else()
+    hunter_add_package(Jpeg)
```

Comment:
This patch should be used https://github.com/hunter-packages/opencv/commit/a9203cdf299efca438631edc62e353093e3ae0ac

Otherwise we will got problems like https://github.com/ruslo/hunter/pull/1613#issuecomment-437308098